packages/react/src/primitives/types/link.ts

```diff
@@ -15,16 +15,6 @@ export interface LinkOptions {
    */
   isExternal?: boolean;
 
-  /**
-   * @deprecated
```

Comment:
I think this prop is mainly created for Next Link `to` type to work correctly but now since we fix `as`, we can get rid of it. Do you think we should keep this prop and remove it in next major release? Feel like we can remove it without waiting for a major bump because we are fixing a bug.

Reply:
I think we probably have to remove it in a major release. Even if it doesn't do anything, it could throw type errors in customer code if removed, right?

Reply:
I am good with that.